front_end/html/view_request.html

```diff
@@ -19,29 +19,31 @@
         </span>
     </div>
     <div class="padded-div">
-    {% for request in similar_requests %}
```

Comment:
small bug fix, I just couched this block of code inside an if statement to check if any similar requests exist first.